packages/create-sitecore-jss/src/templates/nextjs/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["next/babel"]
```

Comment:
What is the purpose of this config? Why it was added?

Reply:
Yeah, this should be removed. We have this as a temporary workaround in our environment/container repo for a Next.js issue, but shouldn't be added here.